pyworklistserver/__main__.py

```diff
@@ -81,20 +97,26 @@ def _get_serverconfig_from_commandline():
         help='Reproduction with last seed.'
     )
 
+    parser.add_argument(
+        '--userconfig',
+        default='./configvalues.json',
+        help='J-son file with configurable values. See user-config.py for available keys.'
+    )
+
     args = parser.parse_args()
     network_address = server_config.NetworkAddress(args.ip, args.port)
-    return (server_config.ServerConfig(network_address, args.aetitle, args.verbose), args.logfile, args.seedfile, args.reproduce)
+    return (server_config.ServerConfig(network_address, args.aetitle, args.verbose), args.logfile, args.seedfile, args.reproduce, args.userconfig)
 
 class PyDicomServer:
     """ Wrapper server implementation that supports stopping through Ctrl+C.
 
     This is just delegating to the core Worklist server implementation
     """
 
-    def __init__(self, config, app_logger, seedfile, reproduce):
+    def __init__(self, config, app_logger, seedfile, reproduce, inserted_config):

```

Comment:
inserted_config is a bit odd name. It is the configuration of the worklist, right? Maybe better to call it worklist_config? 

Also, there is already a config argument. Maybe config should now be renamed server_config, to distinguish the different configs.